### ISSUE TITLE
Installing for all users problem

### ISSUE THREAD

**Issue by zabanet** [yuk7/CentWSL#58]
My user has no admin rights so I can't install Centos7 (error: HRESULT:0xc03a001a). When I install CentOS7.exe as administrator it is installing without issues but is not listed under normal user account using **wsl -l -v**.
Please, make it possible to install it for all users. 
BTW, I have Ubuntu installed on wsl and wsl2 without issues.
Also copying installation files to the user's home directory and installing (still as an administrator because on normal user there was an error) won't do the trick.

**Comment by zabanet** [comment]
I got it! When I found the error description everything cleared out:

ERROR_VIRTUAL_DISK_LIMITATION  
The requested operation could not be completed due to a virtual disk system limitation. Virtual hard disk files must be uncompressed and unencrypted and must not be sparse.

When you install as admin two folders are created in the same directory as your exe (rootfs and tempfs). Right click on each of them as administrator -> Properties -> General -> Advanced and uncheck Compress contents to save disk checkbox. 
It will ask if you want to do it for all the files in the folder. Confirm and skip those which you somehow don't have permission to. After that run the exe file again (this time as normal user) and it will register your distro without issues.
Good job @yuk7! Your work made my life way easier 👍
